### ISSUE TITLE
[freebsd][2014.7.1] traceback when using 'show_timeout: True'

### ISSUE THREAD

**Issue by cedwards** [saltstack/salt#20044]
I activated ```show_timeout: True``` in my master config. When a minion is shutdown I get the following traceback:

```
claudia ~ # salt claudia test.ping
  ckstat: syntax error in port range
[ERROR   ] Failed "sockstat" with returncode = 1
[ERROR   ] An un-handled exception was caught by salt's global exception handler:
CalledProcessError: Command '['sockstat', '-4', '-c', '-p 4505']' returned non-zero exit status 1
Traceback (most recent call last):
  File "/usr/local/bin/salt", line 9, in <module>
    load_entry_point('salt==2014.7.1', 'console_scripts', 'salt')()
  File "/usr/local/lib/python2.7/site-packages/salt/scripts.py", line 241, in salt_main
    client.run()
  File "/usr/local/lib/python2.7/site-packages/salt/cli/__init__.py", line 187, in run
    for full_ret in cmd_func(**kwargs):
  File "/usr/local/lib/python2.7/site-packages/salt/client/__init__.py", line 584, in cmd_cli
    **kwargs):
  File "/usr/local/lib/python2.7/site-packages/salt/client/__init__.py", line 1212, in get_cli_event_returns
    connected_minions = salt.utils.minions.CkMinions(self.opts).connected_ids()
  File "/usr/local/lib/python2.7/site-packages/salt/utils/minions.py", line 444, in connected_ids
    addrs = salt.utils.network.local_port_tcp(int(self.opts['publish_port']))
  File "/usr/local/lib/python2.7/site-packages/salt/utils/network.py", line 897, in local_port_tcp
    ret = remotes_on_local_tcp_port(port)
  File "/usr/local/lib/python2.7/site-packages/salt/utils/network.py", line 1053, in remotes_on_local_tcp_port
    return _freebsd_remotes_on(port, 'local_port')
  File "/usr/local/lib/python2.7/site-packages/salt/utils/network.py", line 1000, in _freebsd_remotes_on
    data = subprocess.check_output(['sockstat', '-4', '-c', '-p {0}'.format(port)])
  File "/usr/local/lib/python2.7/subprocess.py", line 573, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
CalledProcessError: Command '['sockstat', '-4', '-c', '-p 4505']' returned non-zero exit status 1
Traceback (most recent call last):
  File "/usr/local/bin/salt", line 9, in <module>
    load_entry_point('salt==2014.7.1', 'console_scripts', 'salt')()
  File "/usr/local/lib/python2.7/site-packages/salt/scripts.py", line 241, in salt_main
    client.run()
  File "/usr/local/lib/python2.7/site-packages/salt/cli/__init__.py", line 187, in run
    for full_ret in cmd_func(**kwargs):
  File "/usr/local/lib/python2.7/site-packages/salt/client/__init__.py", line 584, in cmd_cli
    **kwargs):
  File "/usr/local/lib/python2.7/site-packages/salt/client/__init__.py", line 1212, in get_cli_event_returns
    connected_minions = salt.utils.minions.CkMinions(self.opts).connected_ids()
  File "/usr/local/lib/python2.7/site-packages/salt/utils/minions.py", line 444, in connected_ids
    addrs = salt.utils.network.local_port_tcp(int(self.opts['publish_port']))
  File "/usr/local/lib/python2.7/site-packages/salt/utils/network.py", line 897, in local_port_tcp
    ret = remotes_on_local_tcp_port(port)
  File "/usr/local/lib/python2.7/site-packages/salt/utils/network.py", line 1053, in remotes_on_local_tcp_port
    return _freebsd_remotes_on(port, 'local_port')
  File "/usr/local/lib/python2.7/site-packages/salt/utils/network.py", line 1000, in _freebsd_remotes_on
    data = subprocess.check_output(['sockstat', '-4', '-c', '-p {0}'.format(port)])
  File "/usr/local/lib/python2.7/subprocess.py", line 573, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
subprocess.CalledProcessError: Command '['sockstat', '-4', '-c', '-p 4505']' returned non-zero exit status 1
```

**Comment by basepi** [comment]
I'm guessing we need to use another command besides `sockstat`, or that it needs different flags on FreeBSD. Thanks for pointing this out.

**Comment by cedwards** [comment]
```sockstat`` is the FreeBSD native command. I'm not sure why it's choking.  I attempted to run the command with the args listed and was successful.

**Comment by basepi** [comment]
Even stranger. Looks like @kev009 may have the fix here. Could you verify, @cedwards?

**Comment by kev009** [comment]
@basepi it looks like a simple args grouping bug for the subprocess call:
'-p{0}' or '-p', {0} would work but '-p {0}' will not.

**Comment by basepi** [comment]
Great catch @kev009 =)

**Comment by basepi** [comment]
I'm pretty certain that that was indeed the problem, so I'm going to go ahead and close this. Keep me posted if it's not resolved.

**Comment by cedwards** [comment]
@basepi - can you tell me whether or not this fix is available in 2014.7.1, or do we need to wait for 2015.2.x?

**Comment by cedwards** [comment]
Actually, it looks like it's still an issue in 2014.7.1. Can you tell me whether the fix is in 2015.2?

**Comment by rallytime** [comment]
@cedwards The fix that is referenced above (#20136) will be included in 2014.7.2, which will be out within the next week. Does more work need to be done here in addition to the pull request above?

**Comment by cedwards** [comment]
@rallytime I don't believe any more work needs to be done. I'm just trying to validate what release will include the fix as I'm trying to get the FreeBSD ports tree updated, and need to know if I can remove the (temporary) dependency on lsof that this addresses.

I will plan to remove the dependency in the 2014.7.2 release. Thanks.
